### ISSUE TITLE
allowing Pipe constructor with more arguments acting as a partial

### ISSUE THREAD

**Issue by YoraiLevi** [JulienPalard/Pipe#93]
The examples in the readme
```python
sum(count() | select(lambda x: x ** 2) | take(10))
```
can also be written as: 
```python
count() | select(lambda x: x ** 2) | take(10) | Pipe(sum)
```
calling the pipe allows using it similarly to partial
```python
count() | select(lambda x: x ** 2) | take(10) | Pipe(sum)(10)
```
The extra brackets is a little odd, I think.

What about this?
```python
count() | select(lambda x: x ** 2) | take(10) | Pipe(sum,10)
```

**Comment by klsmithphd** [comment]
I think this was a great addition. It seems like this issue can be marked as resolved since PR #94 was merged in?

On that note, are there plans to cut a new release soon? I was following the documentation in the `README` and expected to be able to use something akin `Pipe(itertools.accumulate, initial=10)` before I discovered that functionality wasn't available in the latest 2.1 release (where the Pipe constructor still only allows a single `function` argument)
